src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int capacity = 16;
+    private Node<K, V>[] table;
+    private int threshold;
```

Comment:
I don`t think that i need to initialize threshold here or in constructor, becouse i use it in only one method, and if i delete line with initialization it will give the exception.

Reply:
OK, then let it be as it is.